### DISCUSSION TITLE
Please help me with the error.

### DISCUSSION THREAD

**Discussion by izkluxcvy** [catsout/wallpaper-engine-kde-plugin#313] [Q&A]
I think the path to steamlibrary is correct.
What is wrong?
![Screenshot_20230805_033915](https://github.com/catsout/wallpaper-engine-kde-plugin/assets/67828134/aaf7e558-a9fa-47fd-86e1-da7704d49a2b)

**Comment by Impqxr** [comment]
show full logs please
